coalib/bearlib/aspects/collections.py

```diff
@@ -14,3 +15,20 @@ def __contains__(self, aspect):
             if issubaspect(aspect, item):
                 return True
         return False
+
+    def get(self, aspect):
+        """
+        Return first item that match or contain an aspect. See
+        :meth:`coalib.bearlib.aspects.aspectbase.get` for further example.
+
+        :param aspect: An aspectclass OR name of an aspect.
+        :return:       An aspectclass OR aspectclass instance, depend on
+                       AspectList content. Return None if no match found.
+        """
+        if isinstance(aspect, str):
+            aspect = coalib.bearlib.aspects[aspect]
+        try:
+            return next(item.get(aspect) for item in self if
+                        issubaspect(aspect, item))
+        except StopIteration:
+            return None
```

Comment:
`list.get` doesnt exist, but `dict.get` does, and I feel it would be better to use the [`dict.get` semantics](https://docs.python.org/3/library/stdtypes.html#dict.get), which means it should raise `KeyError` here.
@userzimmermann , your thoughts on this?